v2.0/inet.md

```diff
@@ -1,6 +1,86 @@
 ---
 title: INET
-summary:
+summary: The INET data type stores an IPv4 or IPv6 address.
 toc: false
 ---
-<span class="version-tag">New in v2.0:</span> Documentation for the `INET` datatype coming soon!
+<span class="version-tag">New in v2.0:</span> The `INET` [data type](data-types.html) stores an IPv4 or IPv6 address.
+
+<div id="toc"></div>
+
+## Syntax
+
```

Comment:
@nvanbenschoten - how do you determine if a data type can be an interpreted literal, an annotated string literal, or coerced to type INET? Do any of these apply to INET?

Reply:
EDIT: the first one doesn't actually look like it works.. I'll have to fix that during my QA process (https://github.com/cockroachdb/cockroach/issues/22390). I think it's safe to assume that all three will work by the time this is published.

Reply:
Added.